plugin/path_login_test.go

```diff
@@ -34,12 +34,14 @@ func TestLoginIam(t *testing.T) {
 	})
 
 	roleName := "testrole"
+	projects := []string{creds.ProjectId, "someproject"}
 	testRoleCreate(t, b, reqStorage, map[string]interface{}{
 		"name":                   roleName,
 		"type":                   "iam",
 		"policies":               "dev, prod",
-		"project_id":             creds.ProjectId,
```

Comment:
Let's keep/add a test with `project_id`. We don't want to break existing setups/automation that use `project_id`

Reply:
Since there is a test for this in path_role_test.go, I am going to just not keep the deprecated value in the login tests.